lib/app.js

```diff
@@ -205,11 +197,55 @@ module.exports = function(tilelive, options) {
             headers["content-encoding"] = headers["content-encoding"] || "gzip";
           }
 
-          res.set(populateHeaders(headers, params, { 200: true }));
-          return res.send(data);
+          return callback(null, data, populateHeaders(headers, params, { 200: true }));
         });
       });
     });
+  };
+
+  app.get(tilePattern, function(req, res, next) {
+    return getTile(req.params.z | 0, req.params.x | 0, req.params.y | 0,
+        (req.params.scale || "@1x").slice(1, 2), req.params.format,
+        function(status, data, headers) {
```

Comment:
Can you expand the inline variable manipulation into separate variables above this? (For readability)